### ISSUE TITLE
Race condition between htpasswd and admin-user addons

### ISSUE THREAD

**Issue by jorgemoralespou** [minishift/minishift-addons#137]
If one enables htpasswd and admin-user addons, there will be no admin user in the htpasswd file, so it will not be possible to log in as admin to the platform.

**Comment by jorgemoralespou** [comment]
@praveenkumar @LalatenduMohanty @kameshsampath FYI

**Comment by praveenkumar** [comment]
@jorgemoralespou yes that I know so to make it work user need to re-apply the htpasswd addon with admin user and it start working, tested that yesterday :)
